myTrip/trip/models.py

```diff
@@ -3,6 +3,7 @@
 from datetime import datetime
 from django.db import models
 from django.core.exceptions import ObjectDoesNotExist
+from registration.models import CustomUser
```

Comment:
use spaces to separate imports of diff type
https://www.python.org/dev/peps/pep-0008/#imports